tex/gregoriotex.lua

```diff
@@ -327,6 +329,23 @@ local function center_translation(startnode, endnode, ratio, sign, order)
   startnode.head.next.next.kern = -X
 end
 
+local debug_types_activated = {['linesglues'] = false}
+
+local function set_debug_string(debugstring)
+  texio.write_nl(debugstring)
```

Comment:
This line is creating an extra "linesglue" in the log just before the "GregorioTeX is in debug mode..." message.  Do you really need it given that said message already indicates which messages are being printed?